File: CPP/PetShop.cpp
#include <iostream>
#include <string>

using namespace std;

// kelas dasar: PetShop
class PetShop {
protected:
    int id;
    string nama_produk;
    double harga;
    int stok;

public:
    // konstruktor
    PetShop() {}
    PetShop(int id, string nama_produk, double harga, int stok) {
        this->id = id;
        this->nama_produk = nama_produk;
        this->harga = harga;
        this->stok = stok;
    }

    // getter
    int getId() { return id; }
    string getNamaProduk() { return nama_produk; }
    double getHarga() { return harga; }
    int getStok() { return stok; }

    // setter
    void setId(int id) { this->id = id; }
    void setNamaProduk(string nama_produk) { this->nama_produk = nama_produk; }
    void setHarga(double harga) { this->harga = harga; }
    void setStok(int stok) { this->stok = stok; }
};

File: Python/PetShop.py
# kelas dasar: PetShop
class PetShop:
    def __init__(self, id, nama_produk, harga, stok):
        self.id = id
        self.nama_produk = nama_produk
        self.harga = harga
        self.stok = stok

    # Getter
    def get_id(self):
        return self.id

    def get_nama_produk(self):
        return self.nama_produk

    def get_harga(self):
        return self.harga

    def get_stok(self):
        return self.stok